lib/cli/interactive-setup/console-set-org.js

```diff
@@ -0,0 +1,194 @@
+'use strict';
+
+const _ = require('lodash');
+const chalk = require('chalk');
+const { log } = require('@serverless/utils/log');
+const resolveAuthMode = require('@serverless/utils/auth/resolve-mode');
+const apiRequest = require('@serverless/utils/api-request');
+const promptWithHistory = require('@serverless/utils/inquirer/prompt-with-history');
+const { writeOrgAndConsole, showOnboardingWelcome } = require('./utils');
+
+const orgUpdateConfirm = async (stepHistory) => {
+  log.notice(
+    'Service has monitoring enabled, but it is configured with the "org" to which you do not have access'
+  );
+  log.notice();
+  const shouldUpdateOrg = await promptWithHistory({
+    message: 'Would you like to update it?',
+    type: 'confirm',
+    name: 'shouldUpdateOrg',
+    stepHistory,
+  });
+  return shouldUpdateOrg;
+};
+
+const orgsChoice = async (orgNames, stepHistory) => {
+  const orgName = await promptWithHistory({
+    message: 'What org do you want to add this service to?',
+    type: 'list',
+    name: 'orgName',
+    choices: [...orgNames, { name: '[Skip]', value: '_skip_' }],
+    stepHistory,
+  });
+  return orgName;
+};
+
+const resolveOrgNames = async () => {
+  const { userId } = await apiRequest('/me');
+  return new Set((await apiRequest(`users/${userId}/orgs`)).orgs.map(({ orgName }) => orgName));
```

Comment:
```suggestion
  return new Set((await apiRequest(`/users/${userId}/orgs`)).orgs.map(({ orgName }) => orgName));
```

This request fails if we don't have a leading slash

Reply:
Thanks @Danwakeem !